src/main/java/com/cloudbees/jenkins/support/timer/FileListCapComponent.java

```diff
@@ -34,11 +35,11 @@ public void addContents(@NonNull Container container, FileListCap fileListCap) {
             // while we read and put the reports into the support bundle, we don't want
             // the FileListCap to delete files. So we lock it.
 
-            File[] files = fileListCap.getFolder().listFiles(new FilenameFilter() {
+            File[] files = Preconditions.checkNotNull(fileListCap.getFolder().listFiles(new FilenameFilter() {
                 public boolean accept(File dir, String name) {
                     return name.endsWith(".txt");
                 }
-            });
+            }));
             for (File f : Util.fixNull(Arrays.asList(files))) {
```

Comment:
Probably the call to `Util.fixNull` is bogus here. The null would be the return value of `listFiles`, not an _element_ of the return value.